feature-libs/quote/components/list/quote-list.component.html

```diff
@@ -1,111 +1,119 @@
+              placeholder="{{ 'quote.list.sortBy' | cxTranslate }}"
+              [ariaLabel]="'quote.list.sortQuotes' | cxTranslate"
+              ariaControls="quote-list"
+            ></cx-sorting>
+          </label>
+        </div>
+        <table id="quote-list" role="table" aria-describedby="quote-list-desc">
+          <caption id="quote-list-desc" class="cx-visually-hidden">
+            {{
+              'quote.list.title' | cxTranslate
+            }}
+          </caption>
+          <thead role="rowgroup">
+            <tr role="row">
+              <th role="columnheader" aria-sort="none">
                 {{ 'quote.list.name' | cxTranslate }}
-              </div>
-              <span>{{ quote.name }}</span>
-            </td>
-            <td role="cell" class="cx-code">
-              <div class="cx-mobile-header">
+              </th>
+              <th role="columnheader" aria-sort="none">
                 {{ 'quote.header.overview.id' | cxTranslate }}
-              </div>
-              {{ quote.code }}
-            </td>
-            <td role="cell" class="cx-status">
-              <div class="cx-mobile-header">
+              </th>
+              <th role="columnheader" aria-sort="none">
                 {{ 'quote.header.overview.status' | cxTranslate }}
-              </div>
-              <span [ngClass]="getQuoteStateClass(quote.state)">{{
-                'quote.states.' + quote.state | cxTranslate
-              }}</span>
-            </td>
-            <td role="cell" class="cx-updated-date">
-              <div class="cx-mobile-header">
+              </th>
+              <th role="columnheader" aria-sort="none">
                 {{ 'quote.list.updated' | cxTranslate }}
-              </div>
-              {{ quote?.updatedTime | cxDate: dateFormat }}
-            </td>
-            <td role="cell" class="cx-nav-caret">
-              <cx-icon [type]="iconTypes.CARET_RIGHT"></cx-icon>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-      <div class="cx-sort bottom">
-        <div
-          *ngIf="
-            quotesState.data.pagination.totalPages &&
-            quotesState.data.pagination.totalPages > 1
-          "
-        >
-          <cx-pagination
-            [pagination]="quotesState.data.pagination"
-            (viewPageEvent)="changePage($event)"
-          ></cx-pagination>
+              </th>
+              <th></th>
+            </tr>
+          </thead>
+          <tbody role="rowgroup">
+            <tr
+              role="row"
+              #row
+              *ngFor="let quote of quotesState.data.quotes"
+              [routerLink]="
+                {
+                  cxRoute: 'quoteDetails',
+                  params: { quoteId: quote.code }
+                } | cxUrl
+              "
+              (keydown.enter)="row.click()"
+              title="{{ quote.name }}"
+              [attr.aria-label]="'quote.list.clickableRow' | cxTranslate"
```

Comment:
Adding the aria-label to the row leads to only reading this label when the row is selected. The content of the row is not read anymore.
The aria-label should be moved to the cx-icon tag in the last column.